apps/condo/domains/acquiring/schema/PaymentByLinkService.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * Generated by `createservice acquiring.PaymentByLinkService`
+ */
+
+const { get } = require('lodash')
+
+const { createInstance } = require('@open-condo/clients/address-service-client')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/PaymentByLinkService')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const {
+    registerMultiPaymentForOneReceipt,
+    registerMultiPaymentForVirtualReceipt,
+} = require('@condo/domains/acquiring/utils/serverSchema')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const {
+    validateQRCode,
+    BillingReceipt,
+    BillingIntegrationOrganizationContext,
+    BillingRecipient,
+} = require('@condo/domains/billing/utils/serverSchema')
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    ADDRESS_IS_INVALID: {
+        mutation: 'paymentByLink',
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Provided address did not match any FIAS address',
```

Comment:
why FIAS - addressService can work without FIAS id

Reply:
The message was changed